src/dfx/src/lib/nns/install_nns.rs

```diff
@@ -399,19 +417,10 @@ pub async fn download_ic_repo_wasm(
     fs::create_dir_all(wasm_dir)
```

Comment:
Note: download() and download_gz() are the only methods for downloading used in this file.  Making those two methods "lazy" covers all uses.